planemo/galaxy/api.py

```diff
@@ -23,6 +23,19 @@ def gi(port=None, url=None, key=None):
     )
 
 
+def test_credentials_valid(port=None, url=None, key=None, is_admin=False):
+    """Test if provided API credentials are valid"""
+    test_gi = gi(port, url, key)
```

Comment:
http://localhost:8080/api/users/current has an is_admin attribute that might be better - maybe validate against that?

Reply:
Oh yes, that sounds way better